### ISSUE TITLE
Error 1x127 when patching kext

### ISSUE THREAD

**Issue by azrin1972** [BenSova/Patched-Sur#729]
this is the error that I'm getting.
using the laster patch sur 1.0.0 (115)

```
Error 1x127
Message: "/usr/local/lib/Patched-Sur-Patches/Scripts/PatchSystem.sh: line 56: /usr/local/lib/Patched-Sur-Patches/Scripts/../NeededPatches.sh: No such file or directory"
Output: "Checking environment...
[INFO] We're booted into full macOS.
[INFO] Using dirname source.

Confirming patch location...
Set to detect patches, restarting PatchSystem with NeededPatches..."
```

![image](https://user-images.githubusercontent.com/5212844/130363029-f42a5366-cdbc-4ab7-a9c6-bd283e98f8ce.png)

**Comment by Monkiey** [comment]
try this command in terminal:PSPATH=~/.patched-sur
cd $PSPATH && rm -rf Patched-Sur-Patches*
curl -Lo Patched-Sur-Patches.zip https://codeload.github.com/BenSova/Patched-Sur-Patches/zip/8c27158
cd /usr/local/lib && sudo rm -rf Patched-Sur-Patches*
sudo unzip -q $PSPATH/Patched-Sur-Patches.zip
sudo mv Patched-Sur-Patches-8c27158 Patched-Sur-Patches

**Comment by BenSova** [comment]
Better yet...
```
PSPATH=~/.patched-sur
cd $PSPATH && rm -rf Patched-Sur-Patches*
curl -Lo Patched-Sur-Patches.zip https://codeload.github.com/BenSova/Patched-Sur-Patches/zip/e76daf4
cd /usr/local/lib && sudo rm -rf Patched-Sur-Patches*
sudo unzip -q $PSPATH/Patched-Sur-Patches.zip
sudo mv Patched-Sur-Patches-e76daf4 Patched-Sur-Patches
```

**Comment by BenSova** [comment]
After a new installer is created, macOS is updated, or that command is run, the Patch Kexts will no longer fail. Thanks for reporting!

**Comment by azrin1972** [comment]
> Better yet...
> 
> ```
> PSPATH=~/.patched-sur
> cd $PSPATH && rm -rf Patched-Sur-Patches*
> curl -Lo Patched-Sur-Patches.zip https://codeload.github.com/BenSova/Patched-Sur-Patches/zip/e76daf4
> cd /usr/local/lib && sudo rm -rf Patched-Sur-Patches*
> sudo unzip -q $PSPATH/Patched-Sur-Patches.zip
> sudo mv Patched-Sur-Patches-e76daf4 Patched-Sur-Patches
> ```

Thank you @BenSova  & @Monkiey 

after running those command and rerun the patch via Ptched-Sur everything now back to fully working state

Thank you for this great effort
